lib/packages/core/src/typings/api/transactionApi.ts

```diff
@@ -45,4 +45,27 @@ export interface TransactionApi {
         senderPrivateKey: string,
         recipientAddress: string
     ) => Promise<TransactionId | Error>;
+
+
+    /**
+     * Sends a multi-out request to the blockchain
+     *
+     * The transaction will be broadcasted in two steps.
+     * 1. If sameAmount is true. "sendMoneyMultiSame" is called, otherwise "sendMoneyMulti" is called
+     * 2. Take the returned unsigned message and sign it, i.e. the private key won't be transmitted.
+     *
+     * @param transaction The unsigned transaction
+     * @param senderPublicKey The senders public key for sending an _unsigned_ message
+     * @param senderPrivateKey The senders private key to _sign_ the message
+     * @param recipients The multi-out string of recipient->amount key value pairs
+     * @param sameAmount Whether all recipients will receive the same amount
+     * @return The Transaction
+     */
+    sendMoneyMultiOut: (
+        transaction: Transaction,
+        senderPublicKey: string,
+        senderPrivateKey: string,
+        recipients: string,
+        sameAmount: boolean
+    ) => Promise<TransactionId | Error>;
```

Comment:
we are not returning `Error` anymore - see [BurstService](https://github.com/burst-apps-team/phoenix/blob/develop/lib/packages/core/src/burstService.ts#L74)